src/cascade/stats/utils.py

```diff
@@ -0,0 +1,26 @@
+from cascade.dismod.db.metadata import DensityEnum
+
+from cascade.core import getLoggers
+CODELOG, MATHLOG = getLoggers(__name__)
+
+
+def meas_bounds_to_stdev(df):
+    r"""
+    Given data that includes a measurement upper bound and measurement lower
+    bound, assume those are 95% confidence intervals. Convert them to
+    standard error using:
+
+    .. math::
+
+        \mbox{stderr} = \frac{\mbox{upper} - \mbox{lower}}{2 1.96}
```

Comment:
I need to do some work, I can't quite read these.  Does that mean 2*1.96? (which is what I see below)